godel_robots/abb/godel_irb2400/godel_irb2400_moveit_config/config/controllers.yaml

```diff
@@ -1,6 +1,5 @@
 controller_list:
-  - name: ""
-    action_ns: joint_trajectory_action
+  - name: 'joint_trajectory_action'
+    action_ns: ''
```

Comment:
In regards to the rviz question, creating a new file for each riviz configuration might be very inconvenient, unfortunately I'm not sure that there'd be another way around it as rviz doesn't seem to handle namespaces the same way other ros nodes do.  However, as you'll skip the user interaction part then rviz isn't really necessary.  Instead, you'll have a ros node that'd automatically goes through all the steps a human user would, is that correct?